unlock-app/src/components/interface/checkout/CheckoutWrapper.tsx

```diff
@@ -38,9 +36,6 @@ const CheckoutWrapper: React.FunctionComponent<WrapperProps> = ({
       ) : (
         ''
       )}
-      <header>
-        <Title>{icon && <Logo src={icon} />}</Title>
-      </header>
```

Comment:
This is unused in the current app, but causes some extra empty space to appear in the DOM